src/PrestaShopBundle/Form/Admin/Type/ReorderPositionsButtonType.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace PrestaShopBundle\Form\Admin\Type;
+
+use Symfony\Component\Form\Extension\Core\Type\ButtonType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * Class ReorderPositionsButtonType.
+ */
+class ReorderPositionsButtonType extends TranslatorAwareType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->add('position', ButtonType::class, [
+            'label' => $this->trans('Rearrange', 'Admin.Actions'),
+            'row_attr' => ['class' => 'mb-0'],
+            'attr' => [
+                'class' => 'btn-default js-btn-reorder-positions',
+                'data-label-reorder' => $this->trans('Rearrange', 'Admin.Actions'),
+                'data-label-save' => $this->trans('Done', 'Admin.Actions'),
```

Comment:
So if I get the JS right, once you click on the button it refreshes the page with the new sorting? And on reload the button label becomes `Done` ? Isn't it a bit weird 🤔

Reply:
Anyway, with this component, we need to reload the page to do sorting...